### DISCUSSION TITLE
Allure reporter makes it difficult to remove videos of successful tests and leave videos of unsuccessful tests.

### DISCUSSION THREAD

**Discussion by NikGorchichko** [mmisty/cypress-allure-adapter#96] [Q&A] [answered]
The Cypress documentation has an example of updating the configuration file to remove videos of successful test runs.
Here is the link: https://docs.cypress.io/guides/guides/screenshots-and-videos#Delete-videos-for-specs-without-failing-or-retried-tests
I used the sample code with and without the Allure plugin, so I can say that the sample code without the plugin works as predicted. Maybe someone encountered this problem and found a solution?
Thanks to everyone who contributed thoughts to this discussion.

**Comment by mmisty** [comment, marked as answer]
Hello, @NikGorchichko
The problem that you are using `after:spec` event for deleting videos.
Because of this issue https://github.com/cypress-io/cypress/issues/5240 only one event can be registered and I assume your event handler with deleting videos is being overridden by cypress-allure-adapter.

To avoid this (to be able to register several `after:spec` events) you can use workaround mentioned here: 
https://github.com/cypress-io/cypress/issues/5240#issuecomment-1462214696
or here https://github.com/mmisty/cypress-allure-adapter/blob/main/README.pack.md#afterspec-event
